widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetView.kt

```diff
@@ -64,12 +69,14 @@ internal class EntryWidgetView(
         onDismissListener?.invoke()
     }
 
-    private fun disableScrolling() {
-        // Entry Widget might be embedded in scrolling views, disabling the scroll for
-        // itself would prevent unintended scrolling behavior in such cases
-        isNestedScrollingEnabled = false
+    private fun enableScrolling(enable: Boolean) {
+        if (!enable) {
+            // setNestedScrollingEnabled() broke scrolling in some cases (bottom sheet).
+            // By default, it is enabled. So we can only disable it if needed.
+            isNestedScrollingEnabled = false
+        }
```

Comment:
```suggestion
       isNestedScrollingEnabled = enable
```

Reply:
No, `isNestedScrollingEnabled = true` break scrolling up in the bottom sheet fragment. It is not just a setter.